### DISCUSSION TITLE
command: watch &  command: sync

### DISCUSSION THREAD

**Discussion by jandreaucodes** [Taxel/PlexTraktSync#2019] [General]
I'm brand new to this script and have it working perfectly.

Just a quick question: I initially ran the Docker container with `command: sync`, but if I understand things correctly, once I initially sync, I can use `command: watch` instead, as it scrobbles in real-time.

As long as my server stays running, it should always sync to Trakt that way?

**Comment by scoiattolo49** [comment]
I have mine set to run with watch command every time I start the container.
However I periodically (once a day) run a sync, using using the crontab file, to keep Trakt and Plex perfectly aligned. 
Remember that what you do in Trakt (e.g. rating a movie) is not synchronised back to Plex just using watch.
